src/main/java/org/sert2521/gamename/Robot.kt

```diff
@@ -3,19 +3,21 @@ package org.sert2521.gamename
 import edu.wpi.first.wpilibj.IterativeRobot
 import edu.wpi.first.wpilibj.command.Scheduler
 import edu.wpi.first.wpilibj.livewindow.LiveWindow
+import org.sert2521.gamename.Drivetrain.Drivetrain
+import org.strongback.Strongback
 
 /**
  * This is the main robot class which calls various methods depending on the current game stage.
  */
 class Robot : IterativeRobot() {
     override fun robotInit() {
-        info("Robot starting...")
+        Strongback.logger().info("Robot starting...")
         Drivetrain
     }
 
-    override fun autonomousInit() = info("Autonomous starting...")
+    override fun autonomousInit() = Strongback.logger().info("Autonomous starting...")
 
-    override fun teleopInit() = info("Teleop starting...")
+    override fun teleopInit() = Strongback.logger().info("Teleop starting...")
 
     override fun autonomousPeriodic() = Scheduler.getInstance().run()
```

Comment:
I _believe_ we can stop using the Scheduler, too. `Strongback.start()` and `Strongback.submit()` are all we need.